test/utils/utils_log.cpp

```diff
@@ -20,7 +20,8 @@ FILE *mock_fopen(const char *filename, const char *mode) {
 
 const std::string MOCK_FN_NAME = "MOCK_FUNCTION_NAME";
 std::string expected_message = "";
-FILE *expected_stream;
+// The "Logging output not set - logging disabled" message is printed to stderr.
```

Comment:
I'm sorry... what? why is this change here, why wasn't it caught before...? One line up we have `expected_message = ""`

@lplewa for the help, perhaps...? 😉

Reply:
Done: https://github.com/oneapi-src/unified-memory-framework/pull/553